scripting/get5/jsonhelpers.sp

```diff
@@ -91,7 +91,6 @@ stock int AddJsonSubsectionArrayToList(JSON_Object json, const char[] key, Array
       count++;
     }
     array.Cleanup();
-    delete array;
```

Comment:
Are you sure the invalid handle warning you saw came from this? Unless I'm mistaken, sourcepawn doesn't error on deleting a null handle.

Reply:
Yep, I was surprised too

Reply:
I'm still not convinced this is a correct change though. Do you have error logs pointing specifically to an invalid handle at this line?

Reply:
I'll pull the exact error out. But it was definitely an invalid handle.

Reply:
```

Reply:
Looks like the `GetObject` call `array` comes from returns the internal JSON handle, not a copy of it like I thought. Thanks.